### DISCUSSION TITLE
Can I support your locale translation?

### DISCUSSION THREAD

**Discussion by dingoh** [SavedInstances/SavedInstances#793] [General]
Hello.
I am Korean so I use ko-kr localization.

There is missing translations which have been missed for decades or few weeks.
I want to contribute some translation of those.

Actually I tried to make a pull request, but locale file seems very different from Forked project and addon downloaded via Curseforge.
(There is no context in locale file from forked project! but it has only --@localization... code seems dynamically open sort of tables I think.)

I want you allow and tell me how to contribute. (I barely know co-work with git. I use git almost for my own single project so far.)

**Comment by LiangYuxuan** [comment]
You can contribute to localization by submitting them to https://legacy.curseforge.com/wow/addons/saved_instances/localization
